src/common/enums/community.contributors.update.type.ts

```diff
@@ -0,0 +1,4 @@
+export enum CommunityContributorsUpdateType {
+  ADD = 'add',
```

Comment:
Here the action is `add`, in the community service method is `assign`. Better use `assign` for consistency.